code/modules/mob/living/carbon/xenomorph/update_icons.dm

```diff
@@ -16,7 +16,9 @@
 	return FALSE
 
 /mob/living/carbon/xenomorph/update_icons()
-	if(stat == DEAD)
+	if(emoting) //while emoting we pause updates to our icon
+
+	else if(stat == DEAD)
```

Comment:
A) never do empty ifs
B) do this properly with signals instead of snowflake vars